fusequery/query/src/configs/config.rs

```diff
@@ -148,6 +150,9 @@ pub struct Config {
 
     #[structopt(long, env = TLS_SERVER_KEY, default_value = "")]
     pub tls_server_key: String,
+
+    #[structopt(long, env =  DISABLE_REMOTE_CATALOG)]
```

Comment:
What's this config used for?

Reply:
> Add configuration flag disable_remote_catalog to indicate that whether the remote catalog is enabled
So that unit test cases can be executed without bringing up Store services
Ideally, for simple UTs, implementation of a catalog backed by local/in-memory storage is preferred, but that might be too much for a single PR.

Reply:
the default value is `false`, which will enable remote_catalog.
After we have an impl of embedded catalog, this flag could be removed, I think. 
For the time being, it is an ad-hoc flag to make the unit test easier (without starting a store service)